src/components/jobPosting/JobPosting.tsx

```diff
@@ -0,0 +1,32 @@
+import Text from "src/components/text/Text";
+import { CompanyLocation } from "studio/lib/interfaces/companyDetails";
+import { IJobPosting } from "studio/lib/interfaces/jobPosting";
+
+import styles from "./jobPosting.module.css";
+
+interface JobPostingProps {
+  jobPosting: IJobPosting;
+}
+
+export default function JobPosting({ jobPosting }: JobPostingProps) {
+  const jobPostingLocations = jobPosting.locations
+    .map((location: CompanyLocation) => location.companyLocationName)
+    .join(", ");
+
+  return (
+    <a
+      href={jobPosting.recruiteeAdUrl}
+      target="_blank"
+      className={styles.jobPosting}
+    >
+      <div className={styles.details}>
+        <div className={styles.role}>
+          <Text type={"h5"}>{jobPosting.role}</Text>
+        </div>
+        <div className={styles.locations}>
```

Comment:
The location is a Tag component in the design, so I think you should use the component here for consistency ☺️

Reply:
@idamand Yeah, good point! But the `Tag` component expects the type to be either "button" or "link", should we introduce a third type?

Reply:
Ahh the all encompassing Tag component 😅 Perhaps check with Ole Petter if it could be a Badge instead of a Tag since it does not have any of the types the Tag component has? Using a Badge will also be easier with defining the colors!

Reply:
I discussed it with Ole Petter and we decided on using `Badge` instead. See https://github.com/varianter/variant.no/pull/917. 

<img width="1040" alt="Screenshot 2024-12-02 at 14 35 39" src="https://github.com/user-attachments/assets/7cec4b0f-b14b-4d1c-8cc7-bacd50a949d0">